cloud-shell/Dockerfile

```diff
@@ -20,6 +20,9 @@ RUN sudo apt-get install unzip && \
   unzip -o terraform.zip && \
   sudo install terraform /usr/local/bin
 
+# install Postgres client and Json query
+RUN sudo apt-get -y install postgresql-client jq
```

Comment:
jq looks to already be pre-installed on cloudshell instances. Are you sure postgresql-client is needed?

Reply:
`psql` is used to populate DB with initial rating data. it allows to keep the code of the service clean from any deployment and configuration tasks.

Reply:
You are right. `jq` is part of the cloud-shell image. I will remove it in the following PR.